docs/sources/shared/configuration.md

```diff
@@ -3941,6 +3941,25 @@ otlp_config:
 # CLI flag: -limits.block-ingestion-status-code
 [block_ingestion_status_code: <int> | default = 260]
 
+# Block ingestion until the given time for the given policy. Pushes will be
+# assigned to a policy based on the stream matcher configuration. Experimental.
+[block_policy_ingestion_until: <map of string to Time>]
```

Comment:
Are there defaults for any of these settings?  (even if the default is nothing is sest?)
Also, these settings could definitely use an example in the description, since users might not know what these mappings should look like.

Reply:
sorry, I should have closed this PR earlier. I'm dropping it to work on smaller PRs (I already merged one of them). Regarding your suggestion: absolutely. I'll tackle that on my next PRs.